lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -58,6 +58,9 @@ export class CardViewTextItemComponent extends BaseCardView<CardViewTextItemMode
     @Input()
     multiValueSeparator: string = DEFAULT_SEPARATOR;
 
+    @Input()
+    displayLabelForChips: boolean = true;
```

Comment:
I'm not sure I understand, but how it is backward compatible this way? Didn't we say that we would keep the old behaviour as default?

Reply:
Done.